.github/workflows/ci_cd.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     - name: Install Flutter tool
       uses: subosito/flutter-action@v1
       with:
-        channel: 'dev' # or: 'stable' or 'beta'
+        channel: 'stable' # or: 'stable' or 'beta'
```

Comment:
`dev` doesn't have the latest navigator changes yet, `master` didn't work (the flutter action errors when trying to download) but I read that they had cherry picked the changes onto `stable` so I tried and all tests pass.

Reply:
ooh stable :0

Reply:
thanks for the heads up